### PR TITLE
Updated translation files

### PR DESCRIPTION
**Pull request by pmgrove**
:koala:

Updated translation files

/cc @zendesk/quokka @ydrain 

### References
 - Jira link: https://zendesk.atlassian.net/browse/APPS-2354

### Risks
 - None

### CODE REVIEW

**Comment by princemaple** [comment]
:+1:

File: translations/zh-cn.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        "label": "非活动的图标 URL"
+      },
+      "hoverIconURL": {
+        "label": "悬停图标 URL"
+      },
+      "selectedIconURL": {
+        "label": "选中的图标 URL"
+      }
+    },
+    "description": "添加一个自定义图标到侧栏，或嵌入任意网站。以前称为“侧栏图标”app"
```
Comment (princemaple) [inline translations/zh-cn.json]: s/或/并 @ydrain

File: translations/zh-cn.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "app": {
+    "parameters": {
+      "iframeURL": {
+        "label": "Iframe URL"
+      },
+      "inactiveIconURL": {
+        "label": "非活动的图标 URL"
```
Comment (princemaple) [inline translations/zh-cn.json]: s/活动/活跃, maybe

File: translations/zh-cn.json (hunk 1 above)
Comment (pdeuter) [inline translations/zh-cn.json]: @princemaple please file a ticket so that we can investigate properly.  
Thanks,
Paul

**Comment by princemaple** [comment]
created https://zendesk.atlassian.net/browse/OFFAPPS-462 /cc @ydrain @pdeuter

**Comment by ydrain** [comment]
@princemaple Thank you. I will check with the translator.

:+1: 

cc: @zendesk/i18n

File: translations/zh-cn.json (hunk 1 above)
Comment (ydrain) [inline translations/zh-cn.json]: @princemaple 
Confirmed with the translator. Please change this to:
"添加一个自定义图标到侧栏，并嵌入任意网站。以前称为“侧栏图标"

I've update the translation in the DB.

RE: https://localization.zendesk.com/agent/tickets/1508

File: translations/zh-cn.json (hunk 2 above)
Comment (ydrain) [inline translations/zh-cn.json]: No change. https://localization.zendesk.com/agent/tickets/1508

**Pull request merged**